robolectric/src/test/java/org/robolectric/internal/bytecode/InstrumentingClassLoaderTest.java

```diff
@@ -441,78 +440,91 @@ public void shouldInterceptFilteredStaticMethodInvocations() throws Exception {
 
   @Test
   public void byte_shouldBeHandledAsReturnValueFromInterceptHandler() throws Exception {
+    if (InvokeDynamic.ENABLED) return;
```

Comment:
I guess the tests should work both under the old way of shadowing and the one you implemented, right? So maybe it makes sense to run them under both configurations, with `robolectric.invokedynamic.enable=true/false` set (actually, it would be even better if we could inject the way of shadowing in the constructors, but not sure if it is gonna be easy to do that everywhere, so probably not worth it and setting the environment variable is fine). The simplest way of achieving this as I see it would be using a `Parametrized` runner, but maybe you come up with something simpler :)